src/mmrelay/plugins/ping_plugin.py

```
@@ -102,78 +98,69 @@ async def handle_meshtastic_message(
-            is_direct_message = False
-        else:
-            # Some radios omit/zero-fill destination; treat as broadcast to avoid dropping valid pings
-            is_direct_message = False
+        is_direct_message = to_id == my_id
 
         if not self.is_channel_enabled(channel, is_direct_message=is_direct_message):
             return False
 
-        # Log that the plugin is processing the message
         self.logger.info(
             f"Processing message from {longname} on channel {channel} with plugin '{self.plugin_name}'"
```

Ruff (0.15.9)

[warning] Logging statement uses f-string

(G004)